Extended Project 3/public/plantilla.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <title>...:::Name:::...</title>
+    <link rel="stylesheet" href="./styles/styles.css" />
+  </head>
+
+  <body>
+    <header>
+      <p class="name">--------------------Were Wolf---------------------</p>
+    </header>
+    <main>
+      <h1>Jonathan Mendoza</h1>
+    </main>
+    <div class="container">
+      <ul>
+        <li class="question">QUESTION 1</li>
+        <p class="answer">
+          Lorem ipsum dolor sit amet consectetur adipisicing elit. Inventore
+          possimus velit ipsum nulla harum ut dolorem odio in amet voluptas
+          doloribus cum necessitatibus, minima est quae ipsam nobis ex
+          repudiandae.
+        </p>
+        <li class="question">QUESTION2</li>
+        <p class="answer">
+          Lorem ipsum dolor sit amet consectetur adipisicing elit. Inventore
+          possimus velit ipsum nulla harum ut dolorem odio in amet voluptas
+          doloribus cum necessitatibus, minima est quae ipsam nobis ex
+          repudiandae.
+        </p>
+
+        <li class="question">QUESTION 3</li>
+        <p class="answer">
+          Lorem ipsum dolor sit amet consectetur adipisicing elit. Inventore
+          possimus velit ipsum nulla harum ut dolorem odio in amet voluptas
+          doloribus cum necessitatibus, minima est quae ipsam nobis ex
+          repudiandae.
+        </p>
+
```

Comment:
Is this file used in the project?

Reply:
According to Acceptance Criteria, If an extra page is called on the address bar, it must be shown on /public/file.html. So this file was created for testing to be hosted on that route.